### PR TITLE
Add PHN to Profile for AB#10874

### PR DESCRIPTION
**Pull request by sslaws**
# Fixes or Implements [AB#10874](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/10874)

-   [X] Enhancement
-   [ ] Bug
-   [ ] Documentation

## Description

Add the PHN to Profile
Removed package-lock and ran npm install to refresh it.

image

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [X] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

Yes

### Browsers Tested

-   [ ] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

Functional tests will be delivered in another task.
## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 1]
```diff
@@ -78,6 +79,15 @@ export default class ProfileView extends Vue {
     @Action("addError", { namespace: "errorBanner" })
     addError!: (error: BannerError) => void;
 
+    @Action("getPatientData", { namespace: "user" })
```
Comment (tiagograf-AOT) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:82]: It seems you don't need this action and the isPatientLoading getter.

Comment (sslaws) [reply Apps/WebClient/src/ClientApp/src/views/profile.vue:82]: Added the getter to the mounted and removed the isPatienLoading

**Review by tiagograf-AOT — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/profile.vue (hunk 1 above)
Comment (ninosamson) [reply Apps/WebClient/src/ClientApp/src/views/profile.vue:82]: Looks good to me.

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 2]
```diff
@@ -435,6 +443,10 @@ export default class ProfileView extends Vue {
                 this.isLoading = false;
             });
     }
+
+    private get PHN(): string {
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:447]: This should be lowercase: `phn()` rather than `PHN()`.

**Review by ninosamson — APPROVED** [review]
Looks good.

**Review by MikeLyttle — CHANGES_REQUESTED** [review]

**Review by mk08sh — APPROVED** [review]

File: Apps/WebClient/src/ClientApp/src/views/profile.vue (hunk 2 above)
Comment (ninosamson) [reply Apps/WebClient/src/ClientApp/src/views/profile.vue:447]: I hope that one day the margins will be consistent.

File: Apps/WebClient/src/ClientApp/src/views/profile.vue [hunk 3]
```diff
@@ -793,7 +821,8 @@ export default class ProfileView extends Vue {
                                             class="mx-2"
                                             :disabled="tempSMS === smsNumber"
                                             @click="saveSMSEdit()"
-                                            >Save
+                                        >
+                                            Save
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/src/views/profile.vue:825]: this looks like a space was added. Same on line 813 & 797

**Review by mk08sh — APPROVED** [review]

**Review by FuriousLlama — APPROVED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Review by ninosamson — APPROVED** [review]

**Pull request merged**